docs/maps/map-visualizations.asciidoc

```diff
@@ -0,0 +1,12 @@
+[role="xpack"]
+[[maps-aggregations]]
+== Coordinate map and region map visualizations
+
+Before *Elastic Maps*, Kibana provided mapping capabilities with <<tilemap, coordinate map>> and <<regionmap, region map>> visualizations.
+Coordinate map and region map visualizations can only display a single layer from a single index and provide a limited set of styling options.
```

Comment:
I'd be consistent with capitalization. "Coordinate Map" and "Region Map"

Reply:
@gchaps What are your thoughts on the capitalization of "Coordinate Map" and "Region Map" in this docs section. In the create new UI, they are "Coordinate Map" and "Region Map". This docs section does not reference the UI and is just talking about the visualizations in general.

Reply:
lowercase